torchtune/training/__init__.py

```diff
@@ -84,6 +91,13 @@
     "Checkpointer",
     "update_state_dict_for_classifier",
     "ADAPTER_CONFIG",
+    "ADAPTER_CONFIG_FNAME",
```

Comment:
Yeah these can live in individual files or a `constants` file within the training, but let's not expose these please.